layouts/partials/home/cli.html

```diff
@@ -1,9 +1,9 @@
 <div id="carousel-cli" class="text-left">
     <div class="bg-gray-400 rounded-t border-r-2 border-gray-500">
         <div class="flex p-2">
-            <div class="bg-red-600 rounded-full h-3 w-3"></div>
-            <div class="bg-yellow-600 rounded-full h-3 w-3 mx-2"></div>
-            <div class="bg-green-600 rounded-full h-3 w-3"></div>
+            <div class="rounded-full h-3 w-3" style="background-color: #ff605c;"></div>
```

Comment:
Can we get this image updated to this ESC image? https://www.pulumi.com/images/product/esc-screenshot-2.png

Probably should rename it to secure.html instead of cli.html I think.